### PR TITLE
Bump NPD version to v0.7.1 for GCI

### PR DESCRIPTION
**Pull request by wangzhen127**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This PR uses most recent NPD release (v0.7.1) for GCI.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Node-Problem-Detector v0.7.1 is used on GCI.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by wangzhen127** [comment]
/sig node
/priority important-soon
/assign @Random-Liu 
/assign @yguo0905 
/cc @xueweiz

/hold
We should probably make a new NPD release v0.7.1 to include metrics support for custom plugins. Current v0.7.0 does not include that PR.

**Comment by xueweiz** [comment]
By "to include metrics support for custom plugins", you meant #315 right?
I though it's already in 0.7.0?

**Comment by xueweiz** [comment]
(sorry, just realized we are in different repo.
By "315", I meant this:
https://github.com/kubernetes/node-problem-detector/pull/315

**Comment by wangzhen127** [comment]
Ah, my bad, that is included already. I think we can just use v0.7.0 in k8S 1.16.
/hold cancel

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1328,13 +1328,16 @@ function start-node-problem-detector {
...
+
     local -r custom_km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor-counter.json"
     local -r custom_dm_config="${KUBE_HOME}/node-problem-detector/config/docker-monitor-counter.json"
     local -r custom_sm_config="${KUBE_HOME}/node-problem-detector/config/systemd-monitor-counter.json"
+
     flags="${NPD_TEST_LOG_LEVEL:-"--v=2"} ${NPD_TEST_ARGS:-}"
     flags+=" --logtostderr"
-    flags+=" --system-log-monitors=${km_config},${dm_config}"
-    flags+=" --custom-plugin-monitors=${custom_km_config},${custom_dm_config},${custom_sm_config}"
+    flags+=" --config.system-log-monitor=${km_config},${dm_config},${sm_config}"
```
Comment (xueweiz) [inline cluster/gce/gci/configure-helper.sh]: Can we also add a `--config.system-stats-monitor=config/system-stats-monitor.json` ?

Comment (wangzhen127) [reply cluster/gce/gci/configure-helper.sh]: Done

**Comment by wangzhen127** [comment]
/hold
Hold again. In this PR, I updated the E2E test to check KubeletStart event. Currently there is a race condition. On GKE, kubelet is restarted a few times when node is just booted. Those can happen before NPD is started. Without https://github.com/kubernetes/node-problem-detector/pull/319, NPD may not find any kubelet start event if all the kubelet restarts happen before NPD is started.

We should decide if we want NPD to look back from the beginning of the boot time. If yes, we need a new v0.7.1 release to include the fix. If not, we should remove the E2E test update.

**Pull request merged**